pal/src/include/pal/palinternal.h

```diff
@@ -677,4 +677,11 @@ inline T* InterlockedCompareExchangePointerT(
 
 #endif // __cplusplus
 
+#ifndef max
```

Comment:
Curious, why is this needed for fedora and not needed for ubuntu? pal.h also defines min and max, so this seems unnecessary but I saw the same compile error as you. Can you explain why that definition is insufficient?